packages/openactive-broker-microservice/tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "downlevelIteration": true,
     "target": "ES2019",
     "moduleResolution": "node",
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "noImplicitAny": true
```

Comment:
I moved lots of state from state.js to PersistentStore. After making this change, TS didn't show me all the places that needed to be fixed right away because its default functionality (without this flag) is to treat a literal object as being `{ <...allDefinedFields>, [k: string]: any }`. So if you consider `state.asdkfjlasjdflkasjfd`, TS will happily treat it as an `any` 🙃.

I think this is an improvement on the codebase too (it's amazing what TS allows without this), though unfortunately for the reviewer of this PR, it does mean that a lot of the change is just improving various types as was required when turning this on

Reply:
I could not agree more, v type safe!